reference/cd-github.md

```diff
@@ -0,0 +1,92 @@
+---
+title: GitHub Workflow (Preview)
+---
+
+<style>
+    .img-bordered {
+        border: 2px solid rgba(0,0,0,0.15);
+    }
+    .move-right {
+        margin-left: 32px;
+    }
+    .move-right-small {
+        margin-left: 8px;
+    }
+</style>
+
+Pulumi's GitHub workflow integration is a feature which shows the results of Pulumi stack
+updates on GitHub. Showing you any potential infrastructure changes on Pull Requests,
```

Comment:
Nit, the second sentence isn't a complete sentence.

Reply:
Done.